.meteor/local/dev_bundle

```diff
@@ -1 +1 @@
-/home/mitar/.meteor/packages/meteor-tool/1.8.1/mt-os.linux.x86_64/dev_bundle
\ No newline at end of file
+/Users/mikkel/.meteor/packages/meteor-tool/.1.10.2.1xtelwx.8ehw++os.osx.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.osx.x86_64/dev_bundle
```

Comment:
What is this file at all? And this change?

Reply:
I think this file shouldn't be tracked, anything in .meteor/local is a build file, I added a rule in .gitignore to exclude it now

Reply:
I do not think you have to add it to .gitignore. It should be not be committed by default based on `.gitignore` inside `.meteor.` Can you just remove this file and this is it?